examples/fluids/navierstokes.h

```diff
@@ -249,6 +254,62 @@ struct EulerContext_ {
 };
 #endif
 
+// SHOCKTUBE
+#ifndef shocktube_context_struct
+#define shocktube_context_struct
+typedef struct ShockTubeContext_ *ShockTubeContext;
+struct ShockTubeContext_ {
+  CeedScalar Cyzb;
+  CeedScalar Byzb;
+  CeedScalar c_tau;
+  bool implicit;
+  bool yzb;
+  int stabilization;
+};
+#endif
+
+// SHOCKTUBE
+#ifndef shocktube_context_struct
+#define shocktube_context_struct
+typedef struct ShockTubeContext_ *ShockTubeContext;
+struct ShockTubeContext_ {
+  CeedScalar Cyzb;
+  CeedScalar Byzb;
+  CeedScalar c_tau;
+  bool implicit;
+  bool yzb;
+  int stabilization;
+};
+#endif
+
+// SHOCKTUBE
+#ifndef shocktube_context_struct
+#define shocktube_context_struct
+typedef struct ShockTubeContext_ *ShockTubeContext;
+struct ShockTubeContext_ {
+  CeedScalar cv;
+  CeedScalar Cyzb;
+  CeedScalar Byzb;
+  bool implicit;
+  bool yzb;
+  int stabilization;
+};
+#endif
+
+// SHOCKTUBE
+#ifndef shocktube_context_struct
+#define shocktube_context_struct
+typedef struct ShockTubeContext_ *ShockTubeContext;
+struct ShockTubeContext_ {
+  CeedScalar cv;
+  CeedScalar Cyzb;
+  CeedScalar Byzb;
+  bool implicit;
+  bool yzb;
+  int stabilization;
+};
+#endif
+
```

Comment:
These duplicates must not be used
```suggestion
```

Reply:
Thanks, @jedbrown . I thought I removed all the duplicates. Missed this one!